README.rst

```diff
@@ -0,0 +1,71 @@
+===============
+scrapy-zyte-api
+===============
+
+.. image:: https://img.shields.io/pypi/v/scrapy-zyte-api.svg
+   :target: https://pypi.python.org/pypi/scrapy-zyte-api
+   :alt: PyPI Version
+
+.. image:: https://img.shields.io/pypi/pyversions/scrapy-zyte-api.svg
+   :target: https://pypi.python.org/pypi/scrapy-zyte-api
+   :alt: Supported Python Versions
+
+.. image:: https://github.com/scrapy-plugins/scrapy-zyte-api/actions/workflows/test.yml/badge.svg
+   :target: https://github.com/scrapy-plugins/scrapy-zyte-api/actions/workflows/test.yml
+   :alt: Automated tests
+
+.. image:: https://codecov.io/github/scrapinghub/scrapy-zyte-api/coverage.svg?branch=master
+   :target: https://codecov.io/gh/scrapinghub/scrapy-zyte-api
+   :alt: Coverage report
+
+Installation
+------------
+
+.. code-block::
+
+    pip install scrapy-zyte-api
+
+This package requires Python 3.7+.
+
+How to configure
+----------------
+
+Replace the default ``http`` and ``https`` in Scrapy's
+`DOWNLOAD_HANDLERS <https://docs.scrapy.org/en/latest/topics/settings.html>`_.
+
+.. code-block:: python
```

Comment:
Just to be sure, could you please validate the markup using twine, if you haven't done it already? See https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#validating-restructuredtext-markup

Reply:
Yep, it passes locally.

![image](https://user-images.githubusercontent.com/3449761/152339737-c526a7c2-6319-4d36-9e05-daedf73e8cca.png)